File: test/fixtures/output/go/native/query.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	client := &http.Client{}

	url := "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value"

	req, _ := http.NewRequest("GET", url, nil)

	res, _ := client.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: src/targets/node/unirest/fixtures/query.js
const unirest = require('unirest');

const req = unirest('GET', 'http://mockbin.com/har');

req.query({
  foo: [
    'bar',
    'baz'
  ],
  baz: 'abc',
  key: 'value'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});